gaston/second-assignment/index.js

```diff
@@ -0,0 +1,34 @@
+let arrHeroes = [];
+
+function getData() {
+  const url = "/marvel.json";
```

Comment:
```suggestion
  const url = "../../mock/marvel.json";
```

It should wok if you import it like this.

Reply:
I modify the URL just like your suggestion... I was using this url ("/marvel.json") because if i didn't copy the json file on the directory folder i was working, i couldn't make the fetch on the json.